rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -20,23 +20,26 @@ namespace rsx
 			const positionu margin { m_margin, m_margin };
 			positionu pos;
 
+			const auto overlay_width = m_body.get_total_width() + margin.x;
```

Comment:
Margin here is unrelated to `overlay_element`'s margin, it's handled specifically for the perf overlay since we need flexibility for all 4 quadrants (left, top, right, bottom margin).

Just thought I'd include it in this commit because it's kinda related.